src/components/notebook.js

```diff
@@ -3,14 +3,22 @@ import PropTypes from 'prop-types';
 import Iframe from 'react-iframe'
 
 import Typography from '@material-ui/core/Typography';
+import { withStyles } from '@material-ui/core/styles';
 
 import PageHead from './page-head';
 import PageBody from './page-body';
 
-export default class Notebook extends Component {
+const appStyles = theme => ({
+  iframe: {
+    width: '100%',
+    height: '-webkit-fill-available'
```

Comment:
Will this work on non webkit based  browser? And do we care :-) ?

Reply:
It will not, good point. I've updated the styling to calculate from the size of the notebook instead and this seems to work well in both Chrome and Firefox.